packages/core/src/types.ts

```diff
@@ -114,6 +114,13 @@ export type UseFormSetFieldError<Values extends FormValues> = <
   error: FormErrors<PathValue<Values, Name>> | string | string[],
 ) => void;
 
+export type UseFormsSetFieldTouched<Values extends FormValues> = <
```

Comment:
It would be better to use `UseFormSetFieldTouched` instead of `UseFormSetsFieldTouched`

Reply:
done